crates/weaver_resolved_schema/src/body.rs

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(rustdoc::invalid_html_tags)]
+
+//! Specification of a resolved body field.
+
+use crate::attribute::AttributeRef;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use weaver_semconv::attribute::{AttributeType, Examples, RequirementLevel};
+use weaver_semconv::body::{BodyFieldSpec, BodySpec};
+use weaver_semconv::stability::Stability;
+
+/// An attribute definition.
+#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq, Hash, JsonSchema)]
+#[serde(deny_unknown_fields)]
+pub struct Body {
+    /// The body specification.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub fields: Option<Vec<BodyField>>,
+    // Not yet defined in the spec or implemented in the resolver
+    // The body value when there are no fields
+    // #[serde(skip_serializing_if = "Option::is_none")]
+    // pub value: Option<Value>
+}
+
+/// An attribute definition.
+#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq, Hash, JsonSchema)]
+#[serde(deny_unknown_fields)]
+pub struct BodyField {
```

Comment:
Yes, as a `BodyField` is NOT an `Attribute`, it is similar in that there are lots of "common" properties, but a field is missing several of them which will NEVER be used or required. So I defined as an explicit type to ensure that this never occurs or someone does go hey `this` has `xxx` how can I use that (when it will never and have or need it)

Reply:
IIUC though - you DON'T need an unresolved body type yet because body cannot be unresolved right now.  

Also - are we concerned about sharing the same types (`AttributeType`) between these?  Could we evolve without this being a breaking change?

Reply:
Leaving for now, until we see the PR's from ted and we can discuss further.